staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics.go

```diff
@@ -173,7 +185,12 @@ func (m *AdmissionMetrics) ObserveAdmissionController(ctx context.Context, elaps
 }
 
 // ObserveWebhook records admission related metrics for a admission webhook.
-func (m *AdmissionMetrics) ObserveWebhook(ctx context.Context, elapsed time.Duration, rejected bool, attr admission.Attributes, stepType string, extraLabels ...string) {
+func (m *AdmissionMetrics) ObserveWebhook(ctx context.Context, elapsed time.Duration, rejected bool, attr admission.Attributes, stepType string, code int, extraLabels ...string) {
```

Comment:
what's going on with this method signature? it takes var args but all usages only pass in a single argument. from what i can tell, if there were more than one argument passed in the label values would be misaligned in line 194 (or possibly overflow).

Reply:
Yes, this was confusing to me as well. Converted this to a non-variadic function and added the name of the webhook as an arg, similar to `ObserveWebhookRejection`.

Reply:
awesome, thanks for doing that!